Various Improvements

l4rm4nd opened the pull request:
Hi @aress31,

I am using your pentest2xlsx tool quite often and implemented some improvements for testssl parsing over time.

In detail, I've adjusted the following things:

- Updated the `config.py` to include all SSL/TLS vulnerabilities. I noticed that some vulns were missing such as heartbleed. I also adjusted the naming scheme a bit for less confusion about some vulnerabilites (like logjam with export ciphers vs. logjam with weak DH keys only).
- Adjusted the worksheet names for less confusion. Just appended `(Details)` to the sheets holding the detailed scan information. 
- Implemented column autofitting for each worksheet. Who likes doing that manually, innit?
- Implemented conditional formatting (coloring) for specific cell values holding the severity. Who likes doing that manually, innit?
- Updated the `requirements.txt` to hold the openpyxl package, used for autofitting and conditional formatting.
- Some print statements to list skipped testssl json files. May switch to logging again or remove completely. Just helps end users to identify testssl json files that were not parsed successfully.

Just wanted to let you know about the modifications. You may want to implement them, as I find them very helpful.

Cheers

aress31 commented:
Awesome PR, could you kindly share an example output prior to the merging of the PR?

l4rm4nd commented:
Hey @aress31,

I removed the print statements and updated the readme accordingly.

An example XLSX output file can be obtained here:

https://privatebin.net/?c9cd50e321a93254#J5ZYibyMLRoeZdFUaY59zGamdwGgixHmx5uhUMVTHVJv

aress31 commented:
Merged, thanks again @l4rm4nd!
